source/cloud-security/amazon/services/prerequisites/dependencies.rst

```diff
@@ -8,72 +8,22 @@
 Installing dependencies
 =======================
 
-.. note::
-  The integration with AWS S3 can be configured in the Wazuh manager (which also behaves as an agent) or directly in a Wazuh agent. This choice merely depends on how you decide to access your AWS infrastructure in your environment.
-
-.. warning::
-  The Wazuh manager includes all dependencies installed, these steps are only necessary when configuring the integration in a Wazuh agent.
+.. |service| replace:: AWS
 
+.. include:: /_templates/cloud/notes.rst
 
 Python
 ------
 
-The AWS module requires `Python 3 <https://www.python.org/>`__. Specifically, it's compatible with
-`Python |PYTHON_CLOUD_CONTAINERS_MIN|–|PYTHON_CLOUD_CONTAINERS_MAX| <https://www.python.org/downloads/>`_. While later Python versions should work as well, we can't assure they are compatible.
-
-.. tabs::
-
-   .. group-tab:: Yum
-
-      .. code-block:: console
-
-         # yum update && yum install python3
-
-   .. group-tab:: APT
-
-      .. code-block:: console
-
-         # apt-get update && apt-get install python3
-
-
-The required modules can be installed with Pip, the Python package manager. Most UNIX distributions have this tool available in their software repositories:
-
-.. tabs::
-
-   .. group-tab:: Yum
-
-      .. code-block:: console
-
-         # yum update && yum install python3-pip
-
-   .. group-tab:: APT
-
-      .. code-block:: console
-
-         # apt-get update && apt-get install python3-pip
-
-It is recommended to use a pip version greater than or equal to 19.3 to ease the installation of the required dependencies.
-
-.. tabs::
-
-   .. group-tab:: Python 3.8–3.10
-
-      .. code-block:: console
-
-         # pip3 install --upgrade pip
-
-   .. group-tab:: Python 3.11
-
-      .. code-block:: console
-
-         # pip3 install --upgrade pip --break-system-packages
-
-      .. note::
+.. |service_module| replace:: ``AWS``
```

Comment:
We could remove this replacement and simply use `|service|` if they get unified.